File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/ComentarioDAO.java
package pe.edu.pucp.softinv.dao;

import pe.edu.pucp.softinv.model.Comentario.ComentarioDTO;

public interface ComentarioDAO {

    public Integer insertar(ComentarioDTO comentario);

    public ComentarioDTO obtenerPorId(Integer idComentario);

    public Integer modificar(ComentarioDTO comentario);

    public Integer eliminar(ComentarioDTO comentario);

}

File: C#/SoftInvPersistance/DAO/ComentarioDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    public interface ComentarioDAO
    {

        int insertar(ComentarioDTO comentario);

        ComentarioDTO obtenerPorId(int idComentario);

        int modificar(ComentarioDTO comentario);

        int eliminar(ComentarioDTO comentario);

    }
}
